.github/workflows/UnitTests.yml

```diff
@@ -27,28 +27,40 @@ on:
     - cron:  '0 */2 * * *'
 
 jobs:
+
+
   cpu:
-    runs-on: ${{ matrix.os }}
+    name: "CPU test"
+    runs-on: ["self-hosted", "cpu"]
     strategy:
+        fail-fast: false
```

Comment:
put 2 spaces here for indentation of fail-fast

Reply:
Yes, fixed indent now, the test now needs docker installation.